### PR TITLE
Fix Unexpected Functionality for "Maximum Length of Comment" Field

### PR DESCRIPTION
**Pull request by francisco-milan**
This PR fixes the issue documented in #6872

### CODE REVIEW

File: src/ui/lib/zcl_abapgit_html_form_utils.clas.abap [hunk 1]
```diff
@@ -247,7 +247,7 @@ CLASS zcl_abapgit_html_form_utils IMPLEMENTATION.
     DATA:
       lt_fields TYPE zif_abapgit_html_form=>ty_fields,
       lv_value  TYPE string,
-      lv_number TYPE i.
+      lv_number TYPE int8.
```
Comment (mbtools) [inline src/ui/lib/zcl_abapgit_html_form_utils.clas.abap:250]: won't work in 702

**Review by mbtools — CHANGES_REQUESTED** [review]
50 is the recommended max for the title (https://www.git-scm.com/docs/git-commit#_discussion) so that is the default value for the setting.

Users should be able to pick other values. min = 1 is correct, but max should be limited to what fits into a single line. That's about 100 characters. Maybe someone likes higher values and has them already set in abapGit. Let's set max = 250. 

Example where the commit title "should fit":

image

**Comment by francisco-milan** [comment]
Test results after implementing the requested changes:

**Scenario 1**. "Maximum Length of Comment" set to a value <= -10,000,000,000,000,000,000,000,000,000,000 (32 digits or more).
Here I'm proposing to change the original error message, so instead of "Maximum Length of Comment is not numeric" we can display something like: "Input Value Out of Range. Please enter a numeric value with less than 32 digits". 

image

**Scenario 2**. "Maximum Length of Comment" set to a value >= -9,999,999,999,999,999,999,999,999,999,999 AND < 1 (up to 31 digits).

image

**Scenario 3**. "Maximum Length of Comment" set to a value >= 1 AND <= 250.

image

**Scenario 4**. "Maximum Length of Comment" set to a value >= 251 AND <= 9,999,999,999,999,999,999,999,999,999,999 (up to 31 digits).

image

**Scenario 5**. "Maximum Length of Comment" set to a value >= 10,000,000,000,000,000,000,000,000,000,000 (32 digits or more). Also here I would like to use the revised version of the error message instead of the original ("Maximum Length of Comment is not numeric").

image

If you have any questions or comments, please let me know.

Best regards,

Francisco Milan.

**Comment by francisco-milan** [comment]
Closes #6872

**Review by mbtools — APPROVED** [review]
👍

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**